### PR TITLE
Use TRUE and FALSE instead of 1 and 0 for postgres and sqlite

### PR DESCRIPTION
**Pull request by othercorey**
Fixes https://github.com/cakephp/debug_kit/issues/508

This makes the SQL valid when copying to analyze.

### CODE REVIEW

File: src/Database/Log/LoggedQuery.php [hunk 1]
```diff
@@ -73,7 +82,12 @@ protected function interpolate(): string
             if ($p === null) {
                 return 'NULL';
             }
+
             if (is_bool($p)) {
+                if ($this->driver instanceof Postgres || $this->driver instanceof Sqlite) {
```
Comment (ADmad) [inline src/Database/Log/LoggedQuery.php:87]: Wouldn't it be better to check for `Mysql` and return 0/1 and default to `TRUE`/`FALSE`?

Comment (markstory) [reply src/Database/Log/LoggedQuery.php:87]: I think MySQL will also accept TRUE/FALSE.

Comment (othercorey) [reply src/Database/Log/LoggedQuery.php:87]: SqlServer uses `bit` which accepts only 1 or the string 'TRUE' and not the literal TRUE. It was an even split.

Comment (othercorey) [reply src/Database/Log/LoggedQuery.php:87]: Just tested and you are right that MySQL accepts `TRUE` literal.

File: src/Database/Log/LoggedQuery.php [hunk 2]
```diff
@@ -73,8 +81,13 @@ protected function interpolate(): string
             if ($p === null) {
                 return 'NULL';
             }
+
             if (is_bool($p)) {
-                return $p ? '1' : '0';
+                if ($this->driver instanceof Sqlserver) {
+                    return $p ? '1' : '0';
```
Comment (ADmad) [inline src/Database/Log/LoggedQuery.php:87]: Seems we need to update the GA workflow to push coverage report for windows/sqlserver build too.

**Pull request merged**